SYCL/ESIMD/lsc/Inputs/common.hpp

```diff
@@ -11,8 +11,9 @@
 template <int case_num> class KernelID;
 
 template <typename T> T get_rand() {
-  T v = rand();
-  if constexpr (sizeof(T) > 4)
+  using Tuint = sycl::_V1::ext::intel::esimd::detail::uint_type_t<sizeof(T)>;
```

Comment:
I believe 'detail' namespace is for SYCL internal needs only, it should not be used in tests.